src/component/header.tsx

```diff
@@ -0,0 +1,40 @@
+
+import { Box, Flex, Link, Image } from '@chakra-ui/react';
+
+
+
+export const Header = () => {
+
+  return (
+    <Box   p='24px' paddingLeft='80px'>
+    <Flex align="center" >
+      
+      <Link href="/" _hover={{ textDecoration: 'none' }}>
+        <Image src="/src/assets/logoImg.svg" alt="Logo" width="100px" height="48px" />
```

Comment:
Aqui por uma questão de organização, você poderia mudar 
```suggestion
       import LogoImg from '../assets/logoImg.svg'
       
        <Image src={LogoImg} alt="Logo" width="100px" height="48px" />
```

Ajudaria na organização do código.

Reply:
Boa.